vhdl_lang/src/analysis/declarative.rs

```diff
@@ -124,6 +124,7 @@ impl<'a> AnalyzeContext<'a> {
             name,
             subtype_indication,
             signature,
+            info: _,
```

Comment:
I am fine with name changes. Would you prefer another name for `TokenInfo` as well? Unfortunately `TokenSpan` is not possible since the trait is already named that way.

Reply:
In other parts of the code I have used `HasFoo` for traits. So the trait could be called `HasTokenSpan` and the struct `TokenSpan`